src/libxtp/threecenter_gwbse.cc

```diff
@@ -120,13 +120,13 @@ void TCMatrix_gwbse::Fill(const AOBasis& gwbasis, const AOBasis& dftbasis,
     // convolution using the default method.
 #if defined(USE_CUDA)
     if (OPENMP::getThreadId() == 0) {
-      FillBlockCUDA(block, symmstorage, cuda_matrices);
+      block = FillBlockCUDA(block, symmstorage, cuda_matrices);
```

Comment:
nice but can you create the block inside FillBlockCuda so it does not appear as a input parameter

Reply:
i.e 'block=FillBlockCUDA(symmstorage,cuda_matrices)

Reply:
Thanks, fixed in next commit